eng/Versions.props

```diff
@@ -110,8 +110,12 @@
     <MicrosoftVisualStudioTextDataPackageVersion>$(MicrosoftVisualStudioPackagesVersion)</MicrosoftVisualStudioTextDataPackageVersion>
     <MicrosoftVisualStudioTextImplementationPackageVersion>$(MicrosoftVisualStudioPackagesVersion)</MicrosoftVisualStudioTextImplementationPackageVersion>
     <MicrosoftVisualStudioTextLogicPackageVersion>$(MicrosoftVisualStudioPackagesVersion)</MicrosoftVisualStudioTextLogicPackageVersion>
+    <!-- NOTE: The Microsoft.VisualStudio.Threading package version should be kept in sync with O#-Roslyn's version:
+    https://github.com/OmniSharp/omnisharp-roslyn/blob/d7555ebfb6c4d7c6811c58370322d3b092c0abf6/build/Packages.props#L65 -->
```

Comment:
Should there be a target branch linked here instead of the blob?

Reply:
I think GitHub by default now copies line links via absolute commits. This might be so the intended linked line stays the same, as opposed to using a target branch where the line #s may change